sdk/storage/azblob/container/client.go

```diff
@@ -160,6 +160,18 @@ func (c *Client) Delete(ctx context.Context, options *DeleteOptions) (DeleteResp
 	return resp, err
 }
 
+// Restore operation restore the contents and properties of a soft deleted container to a specified container.
+// For more information, see https://docs.microsoft.com/en-us/rest/api/storageservices/restore-container.
+func (c *Client) Restore(ctx context.Context, deletedContainerName string, deletedContainerVersion string) (RestoreResponse, error) {
+	opts := &generated.ContainerClientRestoreOptions{
```

Comment:
We shall redefine this in exported code and that can be used as input options to this API.

Reply:
Done